test/runner.test.ts

```diff
@@ -218,7 +218,7 @@ describe('Runner', () => {
           },
           {
             type: StepType.Click,
-            selectors: [['.parent', '.child']],
+            selectors: ['.parent > .child'],
```

Comment:
this test case was not strictly speaking correct as there is no shadow DOM boundary in the test page => should be a normal CSS selector.